src/hadoop-node-manager/deploy/start.sh

```diff
@@ -28,14 +28,14 @@ pushd $(dirname "$0") > /dev/null
 # Hadoop node manager
 kubectl apply --overwrite=true -f hadoop-node-manager.yaml || exit $?
 
-PYTHONPATH="../.." python -m  k8sPaiLibrary.monitorTool.check_node_label_exist -k hadoop-node-manager -v "true"
+PYTHONPATH="../../../deployment" python -m  k8sPaiLibrary.monitorTool.check_node_label_exist -k hadoop-node-manager -v "true"
```

Comment:
@YitongFeng To ensure, whether this line could be removed or not.

Reply:
could removed